Automattic/wp-calypso
# Plans 2023: add sticky behaviour to the CTA bar

Related to Automattic/martech#1392

## Proposed Changes

* Adds sticky behaviour to the CTA bar on the plans grid. This PR currently does not add the sticky behaviour to the plan comparison grid. 
* Figma: xkhhUlDowF13dRoXJqlRDM-fi-997%3A14697

## Testing Instructions



* Go to `/start/plans` and scroll up. If the viewport height is less, open the comparison grid so there is space to scroll up, and confirm that the sticky header gets activated.
* Once the plans grid is out of view, the CTA bar should no longer stick.
* Confirm that it matches the design.
* When the CTA buttons are "stuck", they should also display the price of the plan. Confirm that the price matches the one shown in the grid.

https://github.com/Automattic/wp-calypso/assets/5436027/ec2873a9-deff-4afe-a08d-b3557f12f5d3

* Repeat the above steps on `/plans/<site slug>`.
* Additionally, the button text when stuck should be "Upgrade - $X"

https://github.com/Automattic/wp-calypso/assets/5436027/14a09f24-8293-480f-a18b-c3d4e2b4cf62

* Repeat the above steps on `/start/launch-site?siteSlug=<site slug>`.
* Additionally, the button text when stuck should be "Select - $X"

<img width="1904" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5436027/417ccaa3-88be-4807-813c-d2830e695261">


## Pre-merge Checklist
